Examples/Modules/nuclear_fusion/inputs_deuterium_deuterium_3d

```diff
@@ -0,0 +1,83 @@
+#################################
+####### GENERAL PARAMETERS ######
+#################################
+## With these parameters, each cell has a size of exactly 1 by 1 by 1
+max_step = 1
+amr.n_cell = 8 8 16
+amr.max_grid_size = 8
+amr.blocking_factor = 8
+amr.max_level = 0
+geometry.dims = 3
+geometry.prob_lo     = 0.    0.    0.
+geometry.prob_hi     = 8.    8.   16.
+
+#################################
+###### Boundary Condition #######
+#################################
+boundary.field_lo = periodic periodic periodic
+boundary.field_hi = periodic periodic periodic
+
+#################################
+############ NUMERICS ###########
+#################################
+warpx.verbose = 1
+warpx.cfl = 1.0
+
+# Order of particle shape factors
+algo.particle_shape = 1
+
+#################################
+############ PLASMA #############
+#################################
+particles.species_names = deuterium_1 hydrogen2_1 helium3_1 neutron_1
```

Comment:
This test is a bit artificial in that it is checking reactions between `deuterium_1` (i.e. a Deuterium species) and `hydrogen2_1` (i.e. also a Deuterium species) and it is not considering intra-species reactions.

I will do a follow-up PR that replaces this with an intra-species test.